### ISSUE TITLE
[BITV] 9.3.3.2/8.3 - All the form fields in the "New user" dialog lack visual labels. Additionally, the required fields are not marked as such. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37091]
See also https://webaim.org/articles/label-name/

- [x] https://github.com/nextcloud/server/pull/38826

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v8-n3

</details>
